Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.py

```diff
@@ -530,7 +539,7 @@ def is_email_valid(email: str) -> bool:
 
 
 @logger
-def test_module(client: Client) -> str:
+def test_module(client: Client, params) -> str:
```

Comment:
did you plan to use it to validate params?

Reply:
Not sure how that was added. I don't remember adding that.
Very strange.
I restored it to how it was initially.